src/service/http-service.js

```diff
@@ -0,0 +1,9 @@
+import axios from 'axios';
+
+const get = (url) => axios.get(url)
+    .then((res) => res.data)
+    .catch(() => undefined);
```

Comment:
this is incorrect
errors should be handled, not ignored

Reply:
okay, how? 
actually, I'm wondering if this line (5) even needs to be here when we have error handling in `load-all-words` action 🤔

Reply:
yeah, now line 5 is like someone saying "Problem?! We don't have any problem at all! Everything is FINE, I repeat everything is FINE" when something went wrong
we don't want such a guy here!